.travis.yml

```diff
@@ -22,9 +22,9 @@ env:
     - WARNINGS_OK=false
 
 jobs:
+  fast_finish: true
   include:
     - env: TEST="clang-format catkin_lint"
-    - env: TEST=code-coverage
```

Comment:
You need to keep the test definition here as well. That's a pitfall of non-intuitive behavior I have tapped in before as well...

Reply:
good catch, put it back here.